### PR TITLE
Added some courses for WS 2015/16

### PR DESCRIPTION
**Pull request by hotzenklotz**
I added some courses for WS 2015/16 and updated the "timestamp" of some older ones.

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -552,12 +553,23 @@ var data = {
...
+    informationIntegration: {
+        kurz: "Information<br /> Integration",
+        kennung: [OSIS, SAMT],
+        lehrform: ["Vorlesung", "Übung"],
+        modul: ["Vertiefungsgebiete"],
+        semester: [ws15_16],
+        benotet: true,
+        cp: 6,
+        dozent: ["Prof. Dr. Felix Naumann"],
+        nameLV: "Information Retrieval"
```
Comment (knub) [inline js/data.js]: Wrong name.

File: js/data.js [hunk 2]
```diff
@@ -552,12 +553,23 @@ var data = {
...
+    },
     informationretrieval: {
         kurz: "Information<br /> Retrieval",
         kennung: [OSIS, IST],
         lehrform: ["Vorlesung", "Übung"],
         modul: ["Vertiefungsgebiete"],
-        semester: [ss14],
+        semester: [ss14, ws15_16],
```
Comment (knub) [inline js/data.js]: Please remove this.
Information Retrieval offers 6 LP this semester, so we need to make a new `informationretrieval6LP`.

File: js/data.js [hunk 3]
```diff
@@ -959,6 +982,17 @@ var data = {
...
     },
+    practicalApplicationsOfMultimediaRetrieval: {
+        kurz: "Practical Applications of <br /> Multimedia Retrieval",
+        kennung: [ITSE, IST, OSIS],
+        lehrform: ["Seminar"],
+        modul: ["Vertiefungsgebiete", "IT-Systems Engineering"],
+        semester: [ws15_16],
+        benotet: true,
+        cp: 6,
+        dozent: ["Prof. Dr. Christoph Meinel", "Haojing Yang "],
```
Comment (knub) [inline js/data.js]: "Haojin"

File: js/data.js [hunk 4]
```diff
@@ -1277,6 +1311,17 @@ var data = {
         dozent: ["Prof. Dr. Holger Giese", "Sebastian Wätzoldt", "Dr. Dominique Blouin"],
         nameLV: "Advanced Software Engineering for Embedded Systems"
     },
+    softwareDesign: {
+        kurz: "Software <br /> Design",
+        kennung: [OSIS, SAMT],
+        lehrform: ["Seminar", "Projekt"],
+        modul: ["Vertiefungsgebiete"],
+        semester: [ws15_16_16],
```
Comment (knub) [inline js/data.js]: `ws15_16_16`

File: js/data.js [hunk 5]
```diff
@@ -1277,6 +1311,17 @@ var data = {
...
+        benotet: true,
+        cp: 6,
+        dozent: ["Prof. Dr. Rober Hirschfeld"],
```
Comment (knub) [inline js/data.js]: "Rober"

File: js/data.js [hunk 6]
```diff
@@ -1381,7 +1426,7 @@ var data = {
         kennung: [ITSE, BPET, OSIS, SAMT],
         lehrform: ["Vorlesung"],
         modul: ["IT-Systems Engineering", "Vertiefungsgebiete"],
-        semester: [ws13_14, ws14_15],
+        semester: [ws13_14, ws14_15, ws15_16],
```
Comment (knub) [inline js/data.js]: Same as above. Can't use this old lecture, because now it offers 6 LP and the specialication areas are different.

**Comment by hotzenklotz** [comment]
@knub I updated the PR and hopefully now everything is correct :-)

**Comment by knub** [comment]
Thanks a lot!

**Pull request merged**